### PR TITLE
Andrei react annotator modified

### PR DESCRIPTION
**Pull request by andrei-msd**

### CODE REVIEW

**Review by yaserazfar — APPROVED** [review]
LGTM. Have tested the functionality with your branch in ufdl-image-annotator. Everything seems to be working.

**Comment by yaserazfar** [comment]
Don't forget to squash before you merge please hahah

File: frontend-react/src/components/annotate.js
Comment (michaeliwaikato) [inline frontend-react/src/components/annotate.js]: This has inconsistent spacing in it (2 vs 4 spaces).

It should probably also be put in a constructor.

File: frontend-react/src/components/annotate.js
Comment (michaeliwaikato) [inline frontend-react/src/components/annotate.js]: It would be good if you could comment this to indicate this is intended to fix an error when the '=' isn't used. (Possibly state the error)

Edit: Probably should take a look at [this](https://reactjs.org/docs/faq-state.html) as it likely fixes some problems
Edits 2: This is definitely a hack and can easily be fixed with a redesign of the logic

File: frontend-react/src/components/annotate.js
Comment (michaeliwaikato) [inline frontend-react/src/components/annotate.js]: This could be causing problems as indicated above.

File: frontend-react/src/components/annotate.js
Comment (michaeliwaikato) [inline frontend-react/src/components/annotate.js]: Same as above.

File: frontend-react/src/components/annotate.js
Comment (michaeliwaikato) [inline frontend-react/src/components/annotate.js]: Probably redundant?

File: frontend-react/src/components/annotate.js
Comment (michaeliwaikato) [inline frontend-react/src/components/annotate.js]: Could cause problems with asynchronous nature as above.

File: frontend-react/src/components/annotate.js
Comment (michaeliwaikato) [inline frontend-react/src/components/annotate.js]: Same again.

File: frontend-react/src/components/annotate.js
Comment (michaeliwaikato) [inline frontend-react/src/components/annotate.js]: And again.

File: frontend-react/src/components/annotate.js
Comment (michaeliwaikato) [inline frontend-react/src/components/annotate.js]: This can fairly easily be shortened and simplified to something like:

`document.getElementById('file').click()`

File: frontend-react/src/components/annotate.js
Comment (michaeliwaikato) [inline frontend-react/src/components/annotate.js]:
```suggestion
      <div id="border">
```

File: frontend-react/src/components/annotate.js
Comment (michaeliwaikato) [inline frontend-react/src/components/annotate.js]:
```suggestion
        <form id="selectForm">
```

Are these forms actually needed anyway?

File: frontend-react/src/components/annotate.js
Comment (michaeliwaikato) [inline frontend-react/src/components/annotate.js]:
```suggestion
            <input type="file" id="file" multiple={true} accept="image/*" onChange={this.fileSelectedHandler} />
```

File: frontend-react/src/components/annotate.js
Comment (michaeliwaikato) [inline frontend-react/src/components/annotate.js]:
```suggestion
```

File: frontend-react/src/components/annotate.js
Comment (michaeliwaikato) [inline frontend-react/src/components/annotate.js]:
```suggestion
        <div id="imageDisplay">
```

File: frontend-react/src/components/annotate.js
Comment (michaeliwaikato) [inline frontend-react/src/components/annotate.js]: Spacing etc should be consistent to make it easier to read/understand.

File: frontend-react/src/components/annotate.js
Comment (michaeliwaikato) [inline frontend-react/src/components/annotate.js]:
```suggestion
    );
  }
```

File: frontend-react/src/components/annotate.js
Comment (michaeliwaikato) [inline frontend-react/src/components/annotate.js]: Same as above in terms of the ids etc.

An idea for this is that this is removed and a base sample image is added to the annotator showing some instructions on how to use. Then the "add image" button is clicked which clears the sample image and shows the users images.

File: frontend-react/src/index.js
Comment (michaeliwaikato) [inline frontend-react/src/index.js:10]: If you can indicate exactly why this was removed then that would be good.

File: frontend-react/src/style/annotate.css
Comment (michaeliwaikato) [inline frontend-react/src/style/annotate.css]:
```suggestion
```

File: frontend-react/src/style/annotate.css
Comment (michaeliwaikato) [inline frontend-react/src/style/annotate.css]: I can't add comments for the parts above, but these should be ids and not classes.

File: frontend-react/src/style/annotate.css
Comment (michaeliwaikato) [inline frontend-react/src/style/annotate.css]: What are these for?

File: frontend-react/src/style/annotate.css
Comment (michaeliwaikato) [inline frontend-react/src/style/annotate.css]: Redundant.

File: frontend-react/src/style/annotate.css
Comment (michaeliwaikato) [inline frontend-react/src/style/annotate.css:46]: Redundant.

File: frontend-react/src/style/annotate.css
Comment (michaeliwaikato) [inline frontend-react/src/style/annotate.css:45]: Seems a little hacky.

File: frontend-react/src/style/annotate.css
Comment (michaeliwaikato) [inline frontend-react/src/style/annotate.css]: Same as above.

**Review by michaeliwaikato — CHANGES_REQUESTED** [review]

**Pull request merged**
